app/config/config.yml

```diff
@@ -60,6 +60,8 @@ twig:
         '%admin_page%/Configure/AdvancedParameters': AdvancedParameters
     globals:
         webpack_server:   false
+    paths:
+        '%admin_page%/Configure/AdvancedParameters': AdvancedParameters
```

Comment:
To be removed, we have a duplicate here.

Reply:
Good catch!